spec/core/validators/validators.spec.tsx

```diff
@@ -18,6 +18,7 @@ describe('Input Validators', () => {
     expect(await Validators.Required(stringWithValue)).toEqual(null);
     expect(await Validators.Required(objectWithValue)).toEqual(null);
     expect(await Validators.Required(arrayWithValues)).toEqual(null);
+    expect(await Validators.Required(new Date())).toEqual(null);
```

Comment:
You could test also this
`expect(await Validators.Required(123546)).toEqual(null);`

Reply:
We could use the library from the React confonrence to generate random values to test :D